docs/sdk/v3/guides/liquidity/02-modifying-position.md

```diff
@@ -0,0 +1,115 @@
+
+All of the liquidity removing logic can be found in the [`removeLiquidity`](https://github.com/Uniswap/examples/blob/d6300e2db41f6a2c3e9c69860347c17c484232ba/v3-sdk/modifying-position/src/example/Example.tsx#L128) function. The function is very similar to `addLiquidity`, except for some details that we will go over. Notice how the **Remove Liquidity** button is disabled until a position is minted.
+
+Note how we do not need to give approval to the `NonfungiblePositionManager` to transfer our tokens as we have already done that when minting our position. 
+
+To start, we create a position identical to the one we created during minting:
+
+```js reference title="Creating an identical position as minting" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/d6300e2db41f6a2c3e9c69860347c17c484232ba/v3-sdk/modifying-position/src/example/Example.tsx#L221
+```
+
+We then pass the new Position, along with an options object of type [`RemoveLiquidityOptions`](https://github.com/Uniswap/v3-sdk/blob/08a7c050cba00377843497030f502c05982b1c43/src/nonfungiblePositionManager.ts#L138) to the `NonfungiblePositionManager`'s `removeCallParameters`, exactly like we did in the adding liquidity case:
+
+```js reference title="Getting the calldata and value for the transaction" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/d6300e2db41f6a2c3e9c69860347c17c484232ba/v3-sdk/modifying-position/src/example/Example.tsx#L224-L238
+```
+Note how we have omitted the `recipient` parameters in the config object, and have instead passed in the `tokenId` of the position we previously minted. `tokenId` is just the `positionId` that we passed in as an argument to the function. In this example, we just pick the last position that we minted:
+
+```js reference title="Passing the tokenId of the last minted position" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/d6300e2db41f6a2c3e9c69860347c17c484232ba/v3-sdk/modifying-position/src/example/Example.tsx#L370-L372
+```
+
+We have also provided two other parameters: `liquidityPercentage` and `collectOptions`. The first is of type `Percentage` and accepts a fraction from 0 to 1, which we provide through our configuration file. This parameter determines how much liquidity is removed from our initial position, and transfers the removed liquidity back to our address. The latter parameter is of type [`CollectOptions`](https://github.com/Uniswap/v3-sdk/blob/08a7c050cba00377843497030f502c05982b1c43/src/nonfungiblePositionManager.ts#L105) and gives us the option to collect the fees that we have collected for the time that we were actively provisioning liquidity. In this example, we pass in 0 for both tokens as we do not want to collect any fees.
```

Comment:
> In this example, we pass in 0 for both tokens as we do not want to collect any fees.

This feels like jumping ahead, but also an odd design we should probably clarify. Why are fees defined in the input and output currencies? Aren't the fees in UNI?